### PR TITLE
Added Keyboard Shortcut Videos to Interface Docs

### LINKED ISSUE #2976: Document keyboard shortcuts
So my original question was

*Is there a keyboard shortcut to navigate between panes in JupyterLab?*

So I went to [the documentation](http://jupyterlab-tutorial.readthedocs.io) and searched for shortcuts and the only link was the "Extensions Developer Guide", a link scary enough to make me turn tail and run :)

Oddly I find that the quick access to the command palette is one of the things that keeps me coming back to JLab.  I suspect that there are other shortcuts that I would enjoy knowing about.  Are these listed someplace?

### PR DESCRIPTION
**Pull request by jzf2101**
Closes #2976

### CODE REVIEW

File: docs/source/user/interface.rst [hunk 1]
```diff
@@ -148,10 +148,18 @@ shortcuts. You can find and customize the current list of keyboard shortcuts by
 selecting the Advanced Settings Editor item in the Settings menu, then selecting
 Keyboard Shortcuts in the Settings tab.
 
-.. image:: images/interface_keyboard_shortcuts.png
```
Comment (ian-r-rose) [inline docs/source/user/interface.rst]: If you are replacing this image, can you also remove it from the repository?

File: docs/source/user/interface.rst [hunk 2]
```diff
@@ -148,10 +148,18 @@ shortcuts. You can find and customize the current list of keyboard shortcuts by
...
-   :class: jp-screenshot
+.. raw:: html
+
+    <div class="jp-youtube-video">
+       <iframe src="https://www.youtube-nocookie.com/embed/rhW3kAExCik?rel=0&amp;showinfo=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+    </div>
 
 You can also customize the text editor to use vim, emacs, or Sublime Text
 keyboard maps by using the Text Editor Key Map submenu in the Settings
 menu.
```
Comment (ian-r-rose) [inline docs/source/user/interface.rst]: Maybe end this sentence with a colon.

File: docs/source/user/interface.rst (hunk 1 above)
Comment (jzf2101) [reply docs/source/user/interface.rst]: Sure

File: docs/source/user/interface.rst (hunk 2 above)
Comment (jzf2101) [reply docs/source/user/interface.rst]: Sounds good

**Comment by jzf2101** [comment]
@ian-r-rose `file_editor.rst` has some of the same content w.r.t. key-maps but i dont think they conflict. I can rework the section or cut and include a link?

**Comment by ian-r-rose** [comment]
Whatever you think is best, I just wanted to flag it.

**Comment by jzf2101** [comment]
Ok I'll just smooth that out.

**Comment by ian-r-rose** [comment]
Thinking about it some more, I am a little puzzled that the keymap discussion is in the `interface.rst`, rather than in `file_editor.rst`. Was there a reason for that?

**Comment by jzf2101** [comment]
There's a broader section for keyboard shortcuts.  We have shortcuts for all of the UI, so we figured the key maps should go there as well.

**Comment by jasongrout** [comment]
Let's merge and iterate. I also think it may be best to put the key maps in the file editor docs, but in a different PR.

**Pull request merged**